app/src/main/java/com/lgallardo/qbittorrentclient/TorrentDetailsFragment.java

```diff
@@ -557,7 +543,7 @@ public void onErrorResponse(VolleyError error) {
                 Map<String, String> params = new HashMap<>();
                 params.put("User-Agent", "qBittorrent for Android");
                 //params.put("Host", hostname + ":" + port);
-                params.put("Referer", protocol + "://" + hostname + ":" + port);
+                params.put("Referer", protocol + "://" + hostname + ( port != -1 ? ":" + port : "" ));
```

Comment:
Perhaps I am missing something, but how do I enter the `-1` value as a port? Or should I just leave it blank? Tapping a minus `-` symbol on the keyboard in the modal for entering the port does not result in entering the symbol:

![image](https://user-images.githubusercontent.com/207112/103406177-1bc57f80-4b5a-11eb-8ce8-45b1b16b33d5.png)

Reply:
Hi @gmile. Leave it blank and it defaults to -1. Then, when building the URL, the port number is left out.